server/lib/routes/post-metrics.js

```diff
@@ -102,6 +109,7 @@ const BODY_SCHEMA = {
   service: STRING_TYPE.regex(SERVICE_PATTERN).required(),
   startTime: TIME_TYPE.required(),
   timers: joi.object().optional(), // this is never consumed.
+  uid: STRING_TYPE.regex(UID_PATTERN).allow('none').required(),
```

Comment:
I'm trying to find where we set `uid` to `none`, what does that? (I'm trying to keep an eye on the unauthenticated metrics events)

Reply:
There's a couple of places in the code added by this PR:

* [On initialisation, if `options.uid` is not set](https://github.com/mozilla/fxa-content-server/pull/5412/files#diff-7b6eb0846bca35324d8b1acd38a32302R132).

* [And in `clearUid`](https://github.com/mozilla/fxa-content-server/pull/5412/files#diff-7b6eb0846bca35324d8b1acd38a32302R651).

Reply:
Thanks @philbooth !